### PR TITLE
[WIP] Add descriptive links for A16y

### PR DESCRIPTION
**Pull request by auchenberg**
Fixes https://github.com/microsoft/vscode-docs/issues/2668

### CODE REVIEW

**Comment by gregvanl** [comment]
@auchenberg I need to think about this. I'm not sure I want us to complicate all links with descriptive text. I agree we shouldn't say "See [here] for more information" but adding a full sentence for descriptive text feels heavy and not standard Markdown.

**Comment by gregvanl** [comment]
@auchenberg I'm going to close this. When I read the MAS 2.2.4 Link Purpose (In Context), I think we're fine just doing a review and making sure we don't have any obvious ambiguous or unclear links. I don't think we want the overhead of providing descriptive text to all links.

**Comment by auchenberg** [comment]
Sounds good @gregvanl

File: docs/python/python-tutorial.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ If you have any problems, feel free to file an issue for this tutorial in the [V
...
+   - (macOS) An installation through [Homebrew](https://brew.sh/ "Homebrew MacOS package manager website") on macOS using `brew install python3` (the system install of Python on macOS is not supported).
+   - (All operating systems) A download from [Anaconda](https://www.anaconda.com/download/ "Anaconda website") (for data science purposes).
 
    > **Note** You can use the `py -0` command in the integrated terminal to view the versions of python installed on your machine. The default interpreter is identified by an asterisk (*).
-1. On MacOS, make sure the location of your VS Code installation is included in your PATH environment variable.  See [the setup instructions](/docs/setup/mac.md#launching-from-the-command-line) for more information.
+1. On MacOS, make sure the location of your VS Code installation is included in your PATH environment variable.  See [the MacOS setup instructions](/docs/setup/mac.md#launching-from-the-command-line) for more information.
 
 ## Start VS Code in a project (workspace) folder
```
Comment (Shay-am-Collab) [inline docs/python/python-tutorial.md]: Sprobuj w powerschel bedziesz mial pewnosc

**Pull request closed without merging**